presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -214,6 +214,7 @@ SystemConfig::SystemConfig() {
           STR_PROP(kExchangeRequestTimeout, "10s"),
           STR_PROP(kExchangeConnectTimeout, "20s"),
           BOOL_PROP(kExchangeEnableConnectionPool, true),
+          BOOL_PROP(kExchangeEnableBufferCopy, true),
```

Comment:
Just confirming you intend to keep this true and not false.

Reply:
Yes.